docs/restic.md

```diff
@@ -29,14 +29,9 @@ cross-volume-type data migrations. Stay tuned as this evolves!
 
 1. Ensure you've [downloaded latest release][3].
 
-1. Install Velero with restic support
+1. Install Velero with restic support:
```

Comment:
let's just delete "Install Velero with restic support:" and move the following sentence up to be the content of #2.

Also - In the note below on RancherOS, can you indent the `path` lines under `hostPath`?